packages/runtime/src/templates/edge-shared/utils.ts

```diff
@@ -6,7 +6,24 @@ export interface FetchEventResult {
   waitUntil: Promise<any>
 }
 
-type NextDataTransform = <T>(data: T) => T
+function normalizeDataUrl(redirect, url) {
```

Comment:
I forgot to mention it on our Zoom, but it'd be good to have a link to where this is done in Next.js code to give more context.

Reply:
That's a great idea, thanks!